src/Utility/Inflector.php

```diff
@@ -625,7 +625,7 @@ public static function underscore($string)
      * Also replaces underscores with dashes
      *
      * @param string $string The string to dasherize.
-     * @return string Dashed version of the input string
+     * @return string|array Dashed version of the input string or multiple strings
```

Comment:
Why do you think this? It is not true. This is still an array.

Reply:
Yes. In cases like having multiple delimiters an array should be passed. As test case given  

```PHP
$this->assertSame('User Is Active', Inflector::humanize('user.is_active',['.','_']));
```
If you pass an array directly you will have a notice error at 
```PHP
 $cacheKey = __FUNCTION__ . $delimiter;
```